Sources/MeiliSearch/Model/IndexesResults.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+/**
+ `IndexesResults` is a wrapper used in the indexes routes to handle the returned data.
+ */
+
+public struct IndexesResults<T: Decodable & Encodable & Equatable>: Codable, Equatable {
```

Comment:
I see that there is a `IndexesResults` but somewhere above you used `Results<Index>`

Reply:
Nice catch, I totally forgot them! :P